specification/maps/data-plane/Creator/preview/2022-09-01-preview/routeset.json

```diff
@@ -0,0 +1,360 @@
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/FacilityDetails"
+          },
+          "x-ms-identifiers": [],
+          "readOnly": true
+        },
+        "created": {
+          "description": "The date and time the routeset object was created.",
+          "type": "string",
+          "format": "date-time",
+          "readOnly": true
+        }
+      }
+    }
+  },
+  "parameters": {
+    "ApiVersion": {
+      "name": "api-version",
+      "description": "Version number of Azure Maps API.",
+      "type": "string",
+      "in": "query",
+      "required": true,
+      "default": "2022-09-01-preview",
+      "x-ms-parameter-location": "client"
+    },
+    "RouteDatasetId": {
+      "name": "datasetId",
+      "type": "string",
+      "in": "query",
+      "description": "The ID of the dataset from which to create the routeset(s). A routeset is created for each facility in the dataset.",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "Description": {
+      "name": "description",
+      "description": "User provided description of the routeset.",
+      "type": "string",
+      "in": "query",
+      "x-ms-parameter-location": "client"
+    },
+    "QueryRoutesetId": {
+      "name": "routesetId",
+      "description": "The ID of the routeset to query for.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "DeleteRoutesetId": {
+      "name": "routesetId",
+      "description": "The ID of the routeset to delete.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    },
+    "OperationId": {
+      "name": "operationId",
+      "description": "The ID used to query the status of the routeset creation.",
+      "type": "string",
+      "in": "path",
+      "required": true,
+      "x-ms-parameter-location": "method"
+    }
+  },
+  "paths": {
+    "/routesets": {
+      "post": {
+        "description": "**Applies to:** see [pricing tiers](https://aka.ms/AzureMapsPricingTier).\n\nCreator makes it possible to develop applications based on your private indoor map data using Azure Maps API and SDK. See the [Creator for indoor maps](https://docs.microsoft.com/azure/azure-maps/creator-indoor-maps) article for an introduction to the Azure Maps Creator concepts and tools.\n\nThis API allows the caller to create routeset data from a dataset.\n\nUse the routeset API to create special data structures consumed by the Azure Maps Wayfinding service to discover the shortest paths within a facility.\n\n### Submit Create Request\n\nTo create a routeset, make a `POST` request with an empty body. The dataset is used as the source of the routeset data, so the `datasetId` query parameter must be included.\n\nThe Create Routeset API is a [long-running request](https://aka.ms/am-creator-lrt-v2).",
+        "operationId": "routesetCreate",
```

Comment:
For SDK, usually this would be "CreateRouteset". But if your team write the SDK, it does not matter.

Reply:
updated to "Routeset_Create" to model what done in our SDK